test/suite/test_schema05.py

```diff
@@ -50,15 +51,18 @@ class test_schema05(wttest.WiredTigerTestCase):
     nentries = 1000
     nindices = 6
 
-    scenarios = make_scenarios([
+    index = [
         ('index-before', { 'create_index' : 0 }),
         ('index-during', { 'create_index' : 1 }),
         ('index-after', { 'create_index' : 2 }),
-    ])
+    ]
+
+    scenarios = make_scenarios(tiered_storage_sources, index)
```

Comment:
Indexes are like column groups and are not expected to work with tiered storage right now.